ui/src/dropdown.tsx

```diff
@@ -59,6 +59,8 @@ export interface Dropdown {
   visible?: B
   /** An optional tooltip message displayed when a user clicks the help icon to the right of the component. */
   tooltip?: S
+  /** Define when the dialog will be displayed. If no value is provided, defaults to 'auto' which will display the dialog only when there are more than 100 choices. */
+  dialog?: 'auto' | 'always' | 'never'
```

Comment:
```suggestion
  popup?: 'auto' | 'always' | 'never'
```

Reply:
I thought of popup as in "the popup window", that's why I used dialog instead. Thinking about it as a verb makes more sense.